geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommandTest.java

```diff
@@ -66,4 +70,14 @@ public void abortIfFileExists() throws IOException {
     gfsh.executeAndAssertThat(command, "export stack-traces --file=" + file.getAbsolutePath())
         .statusIsError().containsOutput("No Members Found");
   }
+
+  @Test
+  public void getHeaderMessage() throws IOException {
+    StackTracesPerMember stackTracePerMember =
+        new StackTracesPerMember("server", Instant.now(),
+            OSProcess.zipStacks());
+    String headerMessage = command.getHeaderMessage(stackTracePerMember);
+
+    assertThat(headerMessage).isNotNull();
+  }
```

Comment:
Thanks for addressing my feedback about the possibility of the timestamp field being null when deserializing a `StackTracesPerMember`. However, I think the test would be more effective if you split it out into a couple of more specific scenarios. For example:
1. When I pass a value of `null` as the timestamp for `StackTracesPerMember`, I expect that the `headerMessage` to be exactly "server". 
2. When I pass a specific value like "2000/01/02 03:04:05.006" as the timestamp for `StackTracesPerMember`, I expect the `headerMessage` to be exactly "server at 2000/01/02 03:04:05.006".

Reply:
Thanks for comments! I added new test with null as the timestamp and modified existing to check if `headerMessage` has specific format as expected in output of this command.

Reply:
Thanks! It looks good.